packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 ## theme-check-vscode
 
+## 3.3.0
+
+### Minor Changes
+
+- 528127bd: Add theme block type completion in `{% schema %}`
+- 5e8a2bfe: Add `SchemaPresetsBlockOrder` check
```

Comment:
Can we copy paste some of the theme check & language server minor changes in here? This changelog is public facing on the VS Code marketplace and could use some manual massaging :) 

![image](https://github.com/user-attachments/assets/33c0813a-24b7-4153-bc3e-03c38d7ba6e7)